### ISSUE TITLE
Stubs do not work with querystrings

### ISSUE THREAD

**Issue by DannyJF** [cheald/manticore#18]
When stubbing a url with a querystring, the stub fails to be invoked:

```
client.stub("http://google.com?k=v", body: "response body", code: 200)
client.get("http://google.com?k=v") do |response|
  response.body.should == "response body"
end
```

**Comment by cheald** [comment]
Looks like this was broken in 717d27eaff32e438c13d6226f829cba795847b30. Patch incoming!

**Comment by cheald** [comment]
Fixed in e2543d62400cfab54ffad9cb4c807e0909222e08. Thanks!
